src/backend/database_migrations/versions/20221122_222651_mpx_hepc_support.py

```diff
@@ -0,0 +1,35 @@
+"""
+
+Create Date: 2022-11-22 22:26:58.178142
+
+"""
+import enumtables  # noqa: F401
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "20221122_222651"
+down_revision = "20221101_201105"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute(
+        """
+        INSERT INTO aspen.pathogens (id, slug, name) VALUES ((SELECT MAX(id)+1 FROM aspen.pathogens), 'HCV', 'Hepatitis C'), ((SELECT MAX(id)+2 FROM aspen.pathogens), 'MPX', 'Monkeypox')
```

Comment:
instead of max(id)+1, please use DEFAULT so PostgreSQL uses its sequences

Reply:
For example, either:
```
insert INTO my_table (id, name) VALUES (DEFAULT, 'bob')
```
or
```
INSERT INTO my_table (name) VALUES ('bob')
```

Basically either of the above queries uses postgresql's default field value, which is attached to a sequence that keeps track of the next available id for our tables.

Reply:
yeah the issue with that is that i accidentally added the ids manually when i did the last insert for the pathogens/pathogen_repo_config tables, I'm working on the solution now.

Reply:
Err, isn't that fixed now? You should be able to update the max value of the sequences if you really have to.

Reply:
Yeah, the default thing works as far as I can tell. In the migration I made, it does it via default (omit setting the id in the SQL) and the result is as expected.

<img width="430" alt="image" src="https://user-images.githubusercontent.com/89553795/203450013-24c2856d-1fa0-4fd9-bdfc-9652f4238adc.png">

https://github.com/chanzuckerberg/czgenepi/blob/rdev-vince-workflow-nextclade/src/backend/database_migrations/versions/20221122_231047_add_mpx_pathogen_and_nextclade_datasets.py

Reply:
sorry i meant i still needed to add it for the pathogen_repo_configs